sharding/collation.go

```diff
@@ -149,29 +129,28 @@ func ConvertBackToTx(rawBlobs []utils.RawBlob) ([]*types.Transaction, error) {
 
 }
 
-// Serialize method  serializes the collation body to a byte array.
-func (c *Collation) Serialize() ([]byte, error) {
-
-	blobs, err := c.CreateRawBlobs()
+// Serialize method serializes the input transactions
+// and returns the chunks in byte arrays.
+func Serialize(txs []*types.Transaction) ([]byte, error) {
```

Comment:
Should this be exported? Should we give it a name that is more specific to sharding? Aside from this, looks good

Reply:
Should be exported for Proposer to serialize TXs and store chunks. Will change to a better name